classes/voice.js

```diff
@@ -27,8 +31,11 @@ class Voice {
 
 module.exports = Voice;
 
-function writeNewSoundFile(filePath, content, callback) {
-    fs.mkdir(ttsDirectory, (err) => fs.writeFile(filePath, content, (err) => callback(err)));
+function writeNewSoundFile(filePath, content, params, callback) {
+	if (params.useGoogleTTS)
```

Comment:
this if could be eliminated if the Voice RSS library is set to also return base 64